### ISSUE TITLE
VS Code settings unrelated to TSLint->ESLint should be ignored

### ISSUE THREAD

**Issue by JoshuaKGoldberg** [typescript-eslint/tslint-to-eslint-config#407]
### 🐛 Bug Report

-   `tslint-to-eslint-config` version: 0.6.0
-   ESLint version: N/A
-   Node version: N/A

#### Actual Behavior

If your `.vscode/settings.json` includes a setting that starts with `editor.`:

```json
{
    "editor.tabSize": 4,
}
```

You'll get a warning when running `tslint-to-eslint-config` that it's not known:

```
1 editor setting is not known by tslint-to-eslint-config to have an ESLint equivalent:
  * tslint-to-eslint-config does not know the ESLint equivalent for TSLint's "editor.tabSize".
```

#### Expected Behavior

Only editor settings relevant to TSLint->ESLints should be complained about.

#### Reproduction

1. Create a directory with a `.vscode/settings.json` equivalent to the above
2. Run `tslint-to-eslint-config`
